light_prop/utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import numpy as np
+from PIL import Image
+
+
+def image_to_amplitude(image_path):
```

Comment:
I'm working in a lang where you can't have functions outside class so its a bit weird looking for me. Do you see any pros/cons with approach you took vs eg. packing it all in something like `ImageLoader/ImageWriter` classes?

Reply:
No function here is sharing any data tbh. They can all be static. But I like to pack stuff in classes just for clarity too, we can do this! Good Idea 👍